sdk/storage/azblob/sas/account.go

```diff
@@ -90,71 +90,6 @@ func (v AccountSignatureValues) SignWithSharedKey(sharedKeyCredential *SharedKey
 	return p, nil
 }
 
-// SignWithUserDelegation uses an account's UserDelegationKey to sign this signature values to produce the proper SAS query parameters.
-func (v AccountSignatureValues) SignWithUserDelegation(userDelegationCredential *UserDelegationCredential) (QueryParameters, error) {
```

Comment:
Was this removed because user delegation can't be used on the account?

Reply:
Yes, user delegation SAS can't be used for account level operations. Please refer the below link.
https://learn.microsoft.com/en-us/rest/api/storageservices/create-user-delegation-sas#specify-permissions